### PR TITLE
grafana-mixin: Fix `GrafanaRequestsFailing` alert

### PR DESCRIPTION
**Pull request by pharaujo**
**What this PR does / why we need it**:
I believe the idea is to get the ratio of 5xx compared will all status code, and to do that, we need to aggregate the `status_code` dimension away.

**Which issue(s) this PR fixes**:
Due to PromQL's label matching, both sides of the division will have same series; that means that, whenever there's a 5xx error, both sides will have the same value and the division will be `1` and the ratio 100%, causing the alert to trigger.

### CODE REVIEW

**Review by joaosilva15 — APPROVED** [review]

File: grafana-mixin/alerts/alerts.yaml [hunk 1]
```diff
@@ -5,8 +5,8 @@ groups:
         for: 5m
         expr: |
           100 * namespace_job_handler_statuscode:grafana_http_request_duration_seconds_count:rate5m{handler!~"/api/datasources/proxy/:id.*|/api/ds/query|/api/tsdb/query", status_code=~"5.."}
-          /
-          namespace_job_handler_statuscode:grafana_http_request_duration_seconds_count:rate5m{handler!~"/api/datasources/proxy/:id.*|/api/ds/query|/api/tsdb/query"}
+          / ignoring (status_code)
+          sum without (status_code) (namespace_job_handler_statuscode:grafana_http_request_duration_seconds_count:rate5m{handler!~"/api/datasources/proxy/:id.*|/api/ds/query|/api/tsdb/query"})
           > 0.5
```
Comment (pharaujo) [inline grafana-mixin/alerts/alerts.yaml:10]: I'm unsure whether this was supposed to be >50% errors; the current >0.5% seems very low for most people. Should I make this configurable with jsonnet?

Comment (bergquist) [reply grafana-mixin/alerts/alerts.yaml:10]: That would be great :)

**Review by bergquist — COMMENTED** [review]
:facepalm:

File: grafana-mixin/alerts/alerts.yaml (hunk 1 above)
Comment (pharaujo) [reply grafana-mixin/alerts/alerts.yaml]: I moved the rules and alerts into `.libsonnet` files. Let me know what you think.

**Review by bergquist — APPROVED** [review]

**Comment by bergquist** [comment]
@pharaujo could you please rebase/merge main?

**Comment by pharaujo** [comment]
@bergquist Done!

**Comment by bergquist** [comment]
Thank you for contributing! :)

**Pull request merged**